src/main/java/org/akaza/openclinica/service/PdfServiceImpl.java

```diff
@@ -260,94 +262,58 @@ public String preparePdfHeader(Study study, Study site, String studySubjectIdent
-           
-           PDFont pdfFont = PDType1Font.TIMES_ROMAN;
-           float fontSize = 10;
-           float leading = 1.5f * fontSize;
-           PDRectangle mediabox = page.getMediaBox();
-           float margin = 50;
-           float width = 140;
-           float startX = mediabox.getLowerLeftX() + margin;
-           float startY = mediabox.getUpperRightY() - margin;
-
-           PDPageContentStream contents = new PDPageContentStream(doc, page);
-           
-           List<String> lines = new ArrayList<String>();
-           int lastSpace = -1;
-           /**
-            * newline character cause some PDF lib methods failed
-            */
-           message = message.replace("\n", "").replace("\r", "");
-           
-           while (message.length() > 0)
-           {
-               int spaceIndex = message.indexOf(' ', lastSpace + 1);
-               if (spaceIndex < 0)
-                   spaceIndex = message.length();
-               String subString = message.substring(0, spaceIndex);
-               float size = subString.length();
-            
-               if (size > width)
-               {
-                   if (lastSpace < 0) {
-                	    lastSpace = spaceIndex;
-                   }
-                      
-                   subString = message.substring(0, lastSpace);
-                   lines.add(subString);
-                   message = message.substring(lastSpace).trim();
-                
-                   lastSpace = -1;
-               }
-               else if (spaceIndex == message.length())
-               {
-                   lines.add(message);                
-                   message = "";
-               }
-               else
-               {
-                   lastSpace = spaceIndex;
-               }
-           }
-
-           contents.beginText();
-           contents.setFont(pdfFont, fontSize);
-           contents.newLineAtOffset(startX,startY);
-           for (String line: lines)
-           {
-        	   contents.showText(line);
-        	   contents.newLineAtOffset(0, -leading);
-           }
-           contents.endText(); 
-           contents.close();
-          
-           
-           doc.save(fileName);
-       }catch(Exception e) {
-    	   logger.error("Error " + e.getMessage());
+       	 file.createNewFile();
+       	 writer = new PrintWriter(file.getPath(), "UTF-8");
+       	 String subStr = "";
+       	 
+       	 // write header in the UI table
+       	 writer.println("SubjectKey,ParticipantID,Status,Timestamp,Message"); 
+       	 
+       	 SimpleDateFormat formatter= new SimpleDateFormat("yyyy-MM-dd HH:mm:ss z");
+       	 Date date = new Date(System.currentTimeMillis());
+       
+       	 writer.println(ss.getOcOid()+","+ss.getLabel()+","+"Failed"+","+formatter.format(date)+","+"Error Detail:");
+       	 
+       	 //during testing, found double quote make JSP error
+       	 msg = msg.replaceAll("\"", "");
```

Comment:
instead of replacing, can you escape the quotes?

Reply:
Done -escape still have same issue, but if write log file in pipe delimited instead of comma delimited will solve this issue.